homeassistant/components/swiss_public_transport/__init__.py

```diff
@@ -1 +1,64 @@
 """The swiss_public_transport component."""
+import logging
+
+from opendata_transport import OpendataTransport
+from opendata_transport.exceptions import OpendataTransportError
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import Platform
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+
+from .const import CONF_DESTINATION, CONF_START, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_START): cv.string,
+        vol.Required(CONF_DESTINATION): cv.string,
+    }
+)
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant, entry: config_entries.ConfigEntry
+) -> bool:
+    """Set up platform from a ConfigEntry."""
+    config = entry.data
+    hass.data.setdefault(DOMAIN, {})
+    hass.data[DOMAIN][entry.entry_id] = config
```

Comment:
Why do we store the config entry data here? This is also being passed on to the async setup entries in the other platforms, so is there any need to store it here?